src/main/java/com/dreammaster/gthandler/recipes/ElectrolyzerRecipes.java

```diff
@@ -14,287 +14,486 @@ public class ElectrolyzerRecipes implements Runnable {
-                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Aluminiumoxide, 9L),
+                Materials.Hydrogen.getGas(10000L),
+                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Iron, 3L),
+                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Aluminiumoxide, 10L),
+                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.SiliconDioxide, 12L),
+                GT_OreDictUnificator.get(OrePrefixes.cell, Materials.Oxygen, 2L),
                 GT_Values.NI,
                 GT_Values.NI,
                 null,
-                944,
+                420,
                 120);
+        // Mica
         GT_Values.RA.addElectrolyzerRecipe(
-                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Aluminiumoxide, 5L),
-                GT_Utility.getIntegratedCircuit(1),
-                GT_Values.NF,
-                Materials.Oxygen.getGas(3000L),
-                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Aluminium, 2L),
-                GT_Values.NI,
+                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Mica, 13L),
                 GT_Values.NI,
+                GT_Values.NF,
+                Materials.Fluorine.getGas(2000L),
+                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Potassium, 1L),
+                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Aluminiumoxide, 15L),
+                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.SiliconDioxide, 9L),
                 GT_Values.NI,
                 GT_Values.NI,
                 GT_Values.NI,
                 null,
-                100,
-                480);
+                260,
+                120);
+        // Alunite
         GT_Values.RA.addElectrolyzerRecipe(
-                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Cryolite, 10L),
-                GT_Utility.getIntegratedCircuit(1),
+                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Alunite, 52L),
+                ItemList.Cell_Empty.get(11L),
                 GT_Values.NF,
-                Materials.Fluorine.getGas(6000L),
-                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Sodium, 3L),
-                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Aluminium, 1L),
+                Materials.Hydrogen.getGas(12000L),
+                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Potassium, 2L),
+                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Aluminiumoxide, 15L),
+                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.SiliconDioxide, 12L),
+                GT_OreDictUnificator.get(OrePrefixes.cell, Materials.Oxygen, 11L),
                 GT_Values.NI,
                 GT_Values.NI,
+                null,
+                210,
+                120);
+        // Zeolite
+        GT_Values.RA.addElectrolyzerRecipe(
+                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Zeolite, 41L),
```

Comment:
18+1+4+27+9 is not 41. dont think there should be oxygen

Reply:
I forgot to fix that recipe, will do

Reply:
The simplest solution I found to this very large chemical formula is to ignore water, aluminium dust instead of alumina